enhancements/ingress/operator-resource-configuration.md

```diff
@@ -0,0 +1,676 @@
+---
+title: ingress-operator-resource-configuration
+authors:
+  - "@joseorpa"
+reviewers:
+  - "TBD"
+  - "TBD"
+  - "TBD"
+  - "TBD"
+approvers:
+  - "TBD"
+api-approvers:
+  - "TBD"
+  - "TBD"
+creation-date: 2025-10-27
+last-updated: 2025-10-27
+tracking-link:
+  - https://issues.redhat.com/browse/RFE-1476
+see-also:
+  - "/enhancements/monitoring/cluster-monitoring-config.md"
+replaces: []
+superseded-by: []
+---
+
+# Ingress Operator Resource Configuration
+
+## Summary
+
+This enhancement proposes adding the ability to configure resource limits and 
+requests for the ingress-operator deployment containers via a new v1alpha1 API 
+field in the IngressController custom resource.
```

Comment:
If this is for the ingress-operator deployment, it doesn't make sense to put this in the IngressController CRD, which describes configuration for router pods.